code/controllers/configuration.dm

```diff
@@ -733,6 +735,9 @@ GLOBAL_LIST_EMPTY(storyteller_cache)
 				if("ruins_start")
 					config.start_location = "space ruins"
 
+				if("profiler_permission")
+					profiler_permission = text2num(value)
```

Comment:
Wasn't the missing config part for `config.profiler_permission` ?

Reply:
![image](https://user-images.githubusercontent.com/46600554/108666885-c29b1c00-750a-11eb-86ee-d1aa5f4382e5.png)
No, there no need to `config.` it's config's proc. I don't know why this whole proc have this, it can broke things if we'll get multiple configs somewhy

Reply:
@AlexMorgan3817 can you elaborate on this please?

Reply:
@AlexMorgan3817 That's the reason. Change it.